lib/ibm_power_hmc/parser.rb

```diff
@@ -932,4 +932,31 @@ def results
       results
     end
   end
+
+  class ManagementConsolePcmPreference < AbstractRest
+    ATTRS = {
+      :max_ltm                     => "MaximumManagedSystemsForLongTermMonitor",
+      :max_compute_ltm             => "MaximumManagedSystemsForComputeLTM",
+      :max_aggregation             => "MaximumManagedSystemsForAggregation",
+      :max_stm                     => "MaximumManagedSystemsForShortTermMonitor",
+      :max_em                      => "MaximumManagedSystemsForEnergyMonitor",
+      :aggregated_storage_duration => "AggregatedMetricsStorageDuration"
+    }
```

Comment:
```suggestion
    }.freeze
```
otherwise rubocop will throw a Style/MutableConstant error.